src/main/scala/com.snowplowanalytics.sauna/responders/urbanairship/UAResponder.scala

```diff
@@ -130,7 +144,15 @@ object UAResponder {
   case class Airship(identifierType: String, identifier: String)
 
   case class UAInput(appKey: String, listName: String, idType: String, id: String)
-
+  
+  type listMap = Map[String, List[Airship]]
```

Comment:
Hey @rmanojcit! 

Type aliases are good. Except that in Java and Scala we have a convention to name types and classes starting with uppercase letter. So, as I wrote earlier, this should be `ListMap`, not `listMap`. That is how we can easily visually distinguish values and types.